package ExceptionHandling;

	public class TryandCatch {

		public static void main(String[] args) {
			 
			        try  
			        {  
			        int number =50;
			        }  
			      
			        catch(ArithmeticException e)  
			        {  
			            System.out.println(e);  
			        }  
			        System.out.println("rest of the code");  
			    }  

		}
